modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -25,6 +25,14 @@ Mule 3.8.5 and above, and all 4.x versions.
 
 ====
 
+== What happens when the transaction limit is reached?
```

Comment:
What happens when the transaction limit for my subscription is reached?

Reply:
Fixed as requested